weaver/typedefs.py

```diff
@@ -220,6 +220,14 @@ class CWL_SchemaName(Protocol):
         "$namespaces": NotRequired[CWL_Namespace],
         "$schemas": NotRequired[Dict[str, str]],
         "$graph": NotRequired[CWL_Graph],
+        "s:author": NotRequired[List[Dict[str, JSON]]],
+        "s:citation": NotRequired[List[Dict[str, str]]],
```

Comment:
Looks like it should be `NotRequired[List[Union[str, JSON]]` based on https://schema.org/citation.